src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/AlwaysEncryptedKeyConverter.CrossPlatform.cs

```diff
@@ -31,7 +75,7 @@ internal sealed class KeyConverter
 
         // Extracts the public key's modulus and exponent from an RSA public key blob
         // and returns an RSAParameters object
-        internal static RSAParameters RSAPublicKeyBlobToParams(byte[] keyBlob)
+        private static RSAParameters RSAPublicKeyBlobToParams(byte[] keyBlob)
```

Comment:
This method is unused now.

Reply:
The same with `ECCPublicKeyBlobToParams` in the same file. I can remove both if you're happy that it's right to do so.

Reply:
Sure, let's remove both. Everything else looks good to me.

Reply:
Done, and fixed a typo in a comment I saw while I was there.